### PR TITLE
Feature/batch plotting

### LINKED ISSUE #43: Make a job for plotting

### PR DESCRIPTION
**Pull request by dylanwells37**
Submits the plotting in a batch job instead of running it in the loop.
Closes #43

### CODE REVIEW

File: BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/physicsOfResultsPUEO.py [hunk 1]
```diff
@@ -32,13 +33,16 @@
...
 
-ROOT.gSystem.Load(libPath + "/libNiceMC.so")
-ROOT.gSystem.Load(libPath + "/libAntarcticaRoot.so")
-ROOT.gSystem.Load(libPath + "/libAnitaEvent.so")
-ROOT.gSystem.Load(libPath + "/libPueoSim.so")
+print(libPath + "/libNiceMC.so")
+ROOT.gSystem.Load("/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/libNiceMC.so")
+ROOT.gSystem.Load("/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/libAntarcticaRoot.so")
+ROOT.gSystem.Load("/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/libAnitaEvent.so")
+ROOT.gSystem.Load("/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/libPueoSim.so")
```
Comment (ezio-melotti) [inline BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/physicsOfResultsPUEO.py:45]: What's the reason for reverting this change?

Comment (dylanwells37) [reply BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/physicsOfResultsPUEO.py:45]: Oh, I was running into errors about files not existing, so I was testing to see if it was causing the issue. It looks like I forgot to revert it back when I found the actual issue - I'll do that now.

File: BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/image_maker.sh [hunk 2]
```diff
@@ -54,11 +53,16 @@ min_index=`cat temp_worst.csv`
 #make sure the correct python is loaded (default 2.7)
 module load python/3.7-2019.10
 module unload python/3.7-2019.10
-source $PSIMDIR/set_env.sh
+source /fs/ess/PAS1960/buildingPueoSim/set_env.sh
```
Comment (Machtay) [inline BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/image_maker.sh:56]: Do we need to hard code the sourcing here? It looks like we're passing the directory variable $PSIMDIR to this script when we call it in Part_F_PUEO.sh.

Comment (dylanwells37) [reply BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/image_maker.sh:56]: Good catch. Forgot to change that back after testing it.

**Pull request merged**